### ISSUE TITLE
Project Review

### ISSUE THREAD

**Issue by albemlee** [zgoldberg93/date-a-scientist#1]
### Criteria 1: Valid Python Code
* _Score Level:_ 4/4
* _Comment(s):_ Code included runs without any errors.
 
### Criteria 2: Exploration of Data
* _Score Level:_ 4/4
* _Comment(s):_ Data is explored significantly, and the experimental question(s) chosen are logical and based on the data exploration. Features chosen to answer the question make sense.

### Criteria 3: Machine Learning Techniques used correctly
* _Score Level:_ 4/4
* _Comment(s):_ Algorithms are used correctly and the correct conclusions are drawn from the results.

Notes:
- Think about the times it takes to train a model and generate a prediction separately. If a model takes a very long time to train but generates predictions quickly, would you choose it over a model that trains quickly but takes longer to generate predictions.
- I recommend checking out sklearn.metrics.classification_report 😉
- You should almost always report recall and precision in addition to accuracy for classification models. Especially given that the data is highly imbalances (as you identified in your report). Would a high accuracy model with low recall be useful?
- You should report both training and test scores to ensure your models did not overfit.

### Criteria 4: Report: Are conclusions clear and supported by data?
* _Score Level:_ 4/4
* _Comment(s):_ Question(s) are stated clearly. The results of 2 regression algorithms and 2 classification algorithms are shown. Conclusions are clearly stated and based on evidence.

### Criteria 5: Code formatting
* _Score Level:_ 3/4
* _Comment(s):_ Code is readable but lacks clarity in formatting.

Note: Add in more headings and explanations for your thought process inside your notebook. For example, where should readers go to find the code cells for your regression analysis.
 
### Overall Score: 19/20
Good job on the project. See my notes above for constructive feedback. In addition, check out [ML interpretability](https://www.kaggle.com/learn/machine-learning-explainability) to take your projects to the next level.
